src/main/java/jp/co/future/uroborosql/SqlUpdateImpl.java

```diff
@@ -19,31 +19,18 @@
  */
 final class SqlUpdateImpl extends AbstractSqlFluent<SqlUpdate> implements SqlUpdate {
 	/** バッチ処理を行うかどうか */
-	private boolean batch = false;
+	private final boolean batch = false;
```

Comment:
@HidekiSugimoto189 
With the removal of the `addBatch` method, this field is no longer needed.
At the same time, there is no need to check the `count` method.

Reply:
@shout-star  
Thanks review.  
This was fixed.